back/boxtribute_server/graph_ql/resolvers.py

```diff
@@ -398,13 +487,90 @@ def resolve_create_qr_code(*_, box_label_identifier=None):
     return create_qr_code(box_label_identifier=box_label_identifier)
 
 
+@mutation.field("changeDistributionEventState")
+@convert_kwargs_to_snake_case
+def resolve_change_distribution_event_state(*_, distribution_event_id, new_state):
+    authorize(permission="distro_event:write")
+    return change_distribution_event_state(distribution_event_id, new_state)
+
+
+@mutation.field("createDistributionEvent")
+@convert_kwargs_to_snake_case
+def resolve_create_distribution_event(*_, creation_input):
+    # TODO: Add permissions
+    authorize(permission="distro_event:write")
+    return create_distribution_event(user_id=g.user.id, **creation_input)
+
+
+@mutation.field("createDistributionSpot")
+@convert_kwargs_to_snake_case
+def resolve_create_distribution_spot(*_, creation_input=None):
+    authorize(permission="location:write")
+    return create_distribution_spot(
+        user_id=g.user.id, distribution_spot_input=creation_input
```

Comment:
would be good to pass `**creation_input` here to unpack the arguments for the `create_distribution_spot` function